packages/node/src/exec/default-attenuator.js

```diff
@@ -83,15 +84,34 @@ export const makeGlobalsAttenuator = (
     if (!policy) {
       return
     }
+
     if (policy === ENDO_POLICY_ITEM_ROOT) {
+      if (rootCompartmentGlobalThis) {
+        throw new ReferenceError(
+          'Root compartment globalThis already initialized; this is a bug'
+        )
+      }
       rootCompartmentGlobalThis = packageCompartmentGlobalThis
-      // ^ this is a little dumb, but only a little - assuming importLocation is called only once in parallel. The thing is - even if it isn't, the new one will have a new copy of this module anyway.
-      // That is unless we decide to use `modules` for passing the attenuator, in which case we have an attenuator defined outside of Endo and we can scope it as we wish. Tempting. Slightly less secure, because if we have a prototype pollution or RCE in the attenuator, we're exposing the outside instead of the attenuators compartment.
+      // ^ this is a little dumb, but only a little - assuming importLocation is
+      // called only once in parallel. The thing is - even if it isn't, the new
+      // one will have a new copy of this module anyway. That is unless we
+      // decide to use `modules` for passing the attenuator, in which case we
+      // have an attenuator defined outside of Endo and we can scope it as we
+      // wish. Tempting. Slightly less secure, because if we have a prototype
+      // pollution or RCE in the attenuator, we're exposing the outside instead
+      // of the attenuators compartment.
       copyWrappedGlobals(originalGlobalThis, packageCompartmentGlobalThis, [
         'globalThis',
         'global',
       ])
     } else {
+      if (!rootCompartmentGlobalThis) {
+        rootCompartmentGlobalThis = new Compartment().globalThis
```

Comment:
(no change requested) 
I was tempted to suggest we only do this in one place at the top and use for root as well so the code could be unified, but that would mean root would not be able to write to it directly and read the polyfilled globals without going through getters.